### DISCUSSION TITLE
Catch all doesn't 404 on non-existing pages?

### DISCUSSION THREAD

**Discussion by notflip** [nuxt/content#2110] [Q&A] [answered]
In my SEO report, I can see that there are the following link that is invalid.

`https://studiomonty.be/blog/cookie-policy/`

Which is correct, because the cookie policy page exists (here: `https://studiomonty.be/cookie-policy/`).

This is my folder structure, is the catch-all method of working causing this? 
Shouldn't Nuxt-Content return a 404?

This is my folder structure

<img width="219" alt="Screenshot 2023-06-17 at 20 23 40" src="https://github.com/nuxt/content/assets/4146384/412b314a-2790-41ad-83c5-c75ce97e34f2">

Any help is welcome, thanks

**Comment by nobkd** [comment, marked as answer]
Your first link displays this text, as the page is not found:
`Document not found, overwrite this content with #not-found slot in <ContentDoc>.`

As you stated, because of the catch-all method, it does not error in a 404.

The [second code block in this section of the docs](https://content.nuxtjs.org/api/components/content-doc#slots) explains how to set content for pages under a catch-all route that aren't found.

**Reply by notflip** [reply]
Thanks, that makes sense
